components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/application.js

```diff
@@ -1,3 +1,39 @@
+$(window).load(function () {
+
+    var client = new SwaggerClient({
+        url: 'https://apis.wso2.com/api/am/store/v0.10/swagger.json',

```

Comment:
Shall we move this SwaggerClient url to a config file. If we do this we will only need to change this location only in one place if we are chaining this URL.